source/interact-data/specify-query.txt

```diff
@@ -0,0 +1,476 @@
+following code:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-combination-ops
+   :end-before: end-logical-combination-ops
+   :language: ruby
+   :dedent:
+
+The ``any_of()``, ``none_of()``, ``nor()``, and ``not()`` operations
+behave similarly.
+
+When you use ``and()``, ``or()``, and ``nor()`` logical operators, they
+operate on the criteria built up to that point:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-logical-combination-ops-2
+   :end-before: end-logical-combination-ops-2
+   :language: ruby
+   :dedent:
+
+not() Behavior
+~~~~~~~~~~~~~~
+
+You can use the ``not()`` method without arguments, in which case it
+negates the next condition that is specified. The ``not()`` method can
+be called with one or more hash conditions or ``Criteria`` objects,
+which are all negated and added to the criteria.
+
+The following examples demonstrate the behavior of ``not()``:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-not-logical
+   :end-before: end-not-logical
+   :language: ruby
+   :dedent:
+
+.. note::
+   
+   You cannot use the ``$not`` operator in MongoDB with a string argument.
+   Mongoid uses the ``$ne`` operator to achieve negation:
+
+   .. literalinclude:: /includes/interact-data/query.rb
+      :start-after: start-not-logical-note
+      :end-before: end-not-logical-note
+      :language: ruby
+      :dedent:
+
+Similarly to ``and()``, the ``not()`` operation negates individual
+conditions for simple field criteria. For complex conditions and when a
+field already has a condition defined on it, {+odm+} emulates ``$not``
+by using an ``{'$and' => [{'$nor' => ...}]}`` construct, since MongoDB
+only supports the ``$not`` operator on a per-field basis rather than
+globally:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-not-behavior
+   :end-before: end-not-behavior
+   :language: ruby
+   :dedent:
+
+If you are using ``not()`` with arrays or regular expressions, view the
+limitations of ``$not`` in the :manual:`{+server-manual+}
+</reference/operator/query/not/>`.
+
+Incremental Query Construction
+------------------------------
+
+By default, when you add conditions to a query, {+odm+} considers each
+condition complete and independent from any other conditions
+present in the query. For example, calling ``in()`` twice adds two separate
+``$in`` conditions:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-incremental-1
+   :end-before: end-incremental-1
+   :language: ruby
+   :dedent:
+
+Some operator methods support building the condition incrementally. In this
+case, when adding a condition which uses one of the supported operators,
+if there already is a condition on the same field using the
+same operator, the operator expressions are combined according to the
+specified *merge strategy*, which is described in the following section.
```

Comment:
S: Shorten and split this since it's pretty dense
```suggestion
Some operator methods support building the condition incrementally. When adding a condition which uses one of the supported operators,
if there already is a condition on the same field using the
same operator, the operator expressions are combined according to the
specified *merge strategy*. The following section describes these merge strategies.
```